rollup.config.js

```diff
@@ -33,7 +34,16 @@ export default [
       }),
       commonJS(),
       json(),
-      postcss({extract: 'dist/webexWidgets.css'}),
+      postcss({
+        extract: 'dist/webexWidgets.css',
+        minimize: true,
+        plugins: [
+          copyAssets({ base: 'dist/assets' })
```

Comment:
Where/when is this `assets` directory generated?

Reply:
The current problem is that the `webexWidgets.css` file contains relative paths to assets (fonts, images), for example `../../fonts/fontName.woff` but none were exported where the package is generated.

The `copyAssets` plugin copies all the assets that he founds in the `webexWidgets.css` file in the `dist` folder (te bo more clear I placed all the assets in the `assets` folder) and rewrites all the paths from the CSS file to point to the location of the new asset. After package build we will have this structure:

```
dist
└── assets
    ├── fonts
    └── images
└── webexWidgets.css
└── webexWidgets.esm.js
```
All the things are happening at the build time when the `build` script is executed.